src/glossary.md

```diff
@@ -0,0 +1,281 @@
+# Fuel Glossary
+
+## Address
+
+An address is a cryptographic hash representing an identity of either a wallet, contract address, or a predicate root.
```

Comment:
I worry about including `contract address` here. Unlike Ethereum, addresses and contract IDs are separate things in Fuel. I suggest removing `contract address` here and creating a separate glossary item for "Contract ID".